ex3/ex3.c

```diff
@@ -9,7 +9,12 @@
 
 int main(void)
 {
-    // Your code here
-
+    fork();
+    if (fork() == 0){
+        printf("hello\n");
+    } else {
+        wait(NULL);
```

Comment:
This `wait(NULL);` is pretty sweet. It's a more general version of `waitpid()`, and is used wonderfully here. 👍